GliaWidgets/ViewController/Survey/Components/Survey.Checkbox.swift

```diff
@@ -59,8 +61,12 @@ extension Survey {
             super.defineLayout()
 
             NSLayoutConstraint.activate([
+
```

Comment:
not neccecary empty line

Reply:
fixed - c9488d9c455449e4749b41dbd5b2d8409e3ba564